packages/embedded-utils/src/types/Space.ts

```diff
@@ -86,6 +88,12 @@ export interface ReusedSpaceWithAccessToken extends BaseSpace {
   spaceInfo?: never
   userInfo?: never
   workbook?: never
+  /**
+   * @deprecated use params up one level instead
+   *
+   * Additional props when creating a Space
+   * Optional
+   */
   spaceBody?: never
```

Comment:
We can't do hard deprecations like this without a major version change. Leave it in and commment it as deprecated for now. The next major version we can review our deprecation comments and remove the actual types

Reply:
I figured we'd leave it in as deprecated as a patch and then remove it in a major.